scripts/subset_crams.py

```diff
@@ -103,13 +107,15 @@ def main(beds: str):
         # -C: output CRAM
         # -L: target regions file, specific to sample
         # --write-index: ...
+        # sets GCS_OAUTH_TOKEN to access data directly
         cram_job.command(
```

Comment:
Probably doesn't make a big difference, but would specifying `@` here allow for some multi-threading? (Assuming the 16G of RAM will result in multiple cores being available.